contracts/ForkableRealityETH_ERC20.sol

```diff
@@ -2,977 +2,173 @@
-        );
-        createTemplate(
-            '{"title": "Should switch to ForkManager %s", "type": "bool"}'
-        );
-        token = _token;
-        if (_question_id != bytes32(0x0)) {
-            _importQuestion(_parent_reality_eth, _question_id);
-        }
-    }
-
-    /// @notice Create a reusable template, which should be a JSON document.
-    /// Placeholders should use gettext() syntax, eg %s.
-    /// @dev Template data is only stored in the event logs, but its block number is kept in contract storage.
-    /// @param content The template content
-    /// @return The ID of the newly-created template, which is created sequentially.
-    function createTemplate(
-        string memory content
-    ) private stateAny returns (uint256) {
-        uint256 id = nextTemplateID;
-        templates[id] = block.number;
-        template_hashes[id] = keccak256(abi.encodePacked(content));
-        emit LogNewTemplate(id, msg.sender, content);
-        nextTemplateID = id + 1;
-        return id;
-    }
-
-    /// @notice Ask a new question without a bounty and return the ID
-    /// @dev Template data is only stored in the event logs, but its block number is kept in contract storage.
-    /// @dev Calling without the token param will only work if there is no arbitrator-set question fee.
-    /// @dev This has the same function signature as askQuestion() in the non-ERC20 version, which is optionally payable.
-    /// @param template_id The ID number of the template the question will use
-    /// @param question A string containing the parameters that will be passed into the template to make the question
-    /// @param arbitrator The arbitration contract that will have the final word on the answer if there is a dispute
-    /// @param timeout How long the contract should wait after the answer is changed before finalizing on that answer
-    /// @param opening_ts If set, the earliest time it should be possible to answer the question.
-    /// @param nonce A user-specified nonce used in the question ID. Change it to repeat a question.
-    /// @return The ID of the newly-created question, created deterministically.
-    function askQuestion(
-        uint256 template_id,
-        string memory question,
-        address arbitrator,
-        uint32 timeout,
-        uint32 opening_ts,
-        uint256 nonce
-    )
-        public
-        returns (
-            // stateNotCreated is enforced by the internal _askQuestion
-            bytes32
-        )
-    {
-        require(templates[template_id] > 0, "template must exist");
-
-        bytes32 content_hash = keccak256(
-            abi.encodePacked(template_id, opening_ts, question)
-        );
-        bytes32 question_id = keccak256(
-            abi.encodePacked(
-                content_hash,
-                arbitrator,
-                timeout,
-                msg.sender,
-                nonce
+    // This arbitrator is assigned automatically when importing questions
```

Comment:
when -> for important governance questions?

Reply:
yes, that's how we plan to use it